Makefile

```diff
@@ -16,7 +16,7 @@ OS ?= $(shell uname -s | tr A-Z a-z)
 ALL_ARCH = amd64 arm arm64 ppc64le s390x
 PKG = github.com/prometheus/common
 PROMETHEUS_VERSION = 2.42.0
-GO_VERSION = 1.20.2
+GO_VERSION = 1.20.3
```

Comment:
go 1.20.4 got released yesterday, please use this version
https://groups.google.com/g/golang-announce/c/MEb0UyuSMsU

Reply:
Updated.

Reply:
It seems like it is still at 1.20.3, can you push the change?